lib/project_types/script/forms/ask_app.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Script
+  module Forms
+    class AskApp < ShopifyCLI::Form
+      attr_accessor :app
+      
+      def ask
+        apps = @xargs
+        unless ShopifyCLI::Shopifolk.acting_as_shopify_organization?
```

Comment:
This is a query into the infra layer again.  In this case, we can pass it as a boolean parameter representing the value from the application layer.  Like `ask(acting_as_shopify_organization)`.